packages/components/ng-at-internet/src/constants.js

```diff
@@ -33,6 +33,11 @@ export const AT_INTERNET_CUSTOM_VARS = {
     path: 'site.6', // OVH's AtInternet configuration
     format: '[%s]',
   },
+
+  siteName: {
+    path: 'site.13', // OVH's AtInternet configuration
```

Comment:
Not yours but I think we can take this opportunity to rename `OVH` into `OVHcloud` 😎.